main/init_css.c

```diff
@@ -0,0 +1,184 @@
+#include "css-generator.h"
+#include "html-generator.h"
+#include "init-css.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+void initCSS(CSS *css) {

```

Comment:
Function name collision in `css-generator.h`. Rename this function to `createCSS`. Rename files to `create-css.c` and `create-css.h` too.